lib/js/components/Form/CheckboxGroupField/CheckboxGroupField.vue

```diff
@@ -0,0 +1,101 @@
+<template>
+	<form-field v-bind="formFieldProps">
+		<template #field="{ messageId, labelId }">
+			<checkbox-group-root
```

Comment:
this component accepts `disabled` prop. I didn't change it's functionality but sounds like a safe option to be set. Maybe if not now it'll add some accesibility features in future